src/twisted/test/test_sslverify.py

```diff
@@ -288,17 +289,27 @@ def connectionLost(self, reason):
     plainServerFactory = protocol.Factory()
     plainServerFactory.protocol = lambda: serverWrappedProto
 
+    clock = Clock()
     clientFactory = TLSMemoryBIOFactory(
-        clientOpts, isClient=True, wrappedFactory=plainServerFactory
+        clientOpts, isClient=True, wrappedFactory=plainServerFactory, clock=clock
     )
     serverFactory = TLSMemoryBIOFactory(
-        serverOpts, isClient=False, wrappedFactory=plainClientFactory
+        serverOpts, isClient=False, wrappedFactory=plainClientFactory, clock=clock
     )
 
     sProto, cProto, pump = connectedServerAndClient(
         lambda: serverFactory.buildProtocol(None),
         lambda: clientFactory.buildProtocol(None),
     )
+
+    # Need time to pass for flushing to work:
+    def flush(pump_flush=pump.flush):
+        clock.advance(0.001)
```

Comment:
Does advancing 0.0 not work? It'd be preferable to avoid disrupting tests that rely on advancing time in precise ways (I've written such tests in the past to cover timeout/retry logic).

Reply:
Hm. Looking at the implementation, it might actually.